app/views/applicants/_form.html.haml

```diff
@@ -11,8 +11,9 @@
 
     = form.inputs name: t("applicants.forms.register.login_information") do
       != form.input :email, input_html: { value: @applicant_login_email }
-      != form.input :password, input_html: {value: ""}
-      != form.input :password_confirmation, input_html: {value: ""}
+      != form.input :email_confirmation, input_html: { value: "" }
```

Comment:
Ser ut til at input-feltet blir av type `text`, og for en eller annen grunn når registreringen feiler, blir stylingen litt whack. Dette ser ut til å fikse det.

```suggestion
      != form.input :email_confirmation, input_html: { type: "email", value: "" }
```